modules/mod-deep-learning/ExploreHuggingFaceDialog.cpp

```diff
@@ -0,0 +1,62 @@
+/**********************************************************************
+
+   Audacity: A Digital Audio Editor
+   Audacity(R) is copyright (c) 1999-2021 Audacity Team.
+
+   ExploreHuggingFaceDialog.cpp
+   Hugo Flores Garcia
+
+******************************************************************/
+
+#include "ExploreHuggingFaceDialog.h"
+
+#include "Shuttle.h"
+#include "ShuttleGui.h"
+
+#include <wx/scrolwin.h>
+#include <wx/range.h>
+#include <wx/gauge.h>
+#include <wx/button.h>
+#include <wx/sizer.h>
+#include <wx/textdlg.h>
+#include <wx/stattext.h>
+#include <wx/hyperlink.h>
+
+// ExploreDialog
+
+ExploreHuggingFaceDialog::ExploreHuggingFaceDialog(wxWindow *parent)
+   : wxDialogWrapper(parent, wxID_ANY, XO("Explore Models"))
```

Comment:
Might precede this line with
`//i18n-hint <explanation of "Models" for the translators>`
I believe Dmitry has already emphasized to you the importance of the special i18n-hint comments, which get extracted from the source into the translation catalog, which human translation experts must fill in.  Unusual words out of context might need some more helpful explanation.

Reply:
resolved in 09844aa67

Reply:
resolved in 09844aa67